docs/testing/cypress.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Testing with Cypress
+description: Use Cypress to write end-to-end tests with Clerk.
+---
+
+# Testing with Cypress
+
+<Callout type="info">
```

Comment:
```suggestion
<Callout type="warning">
```
❓ Should we change that to a warning to be a bit more obvious?

Reply:
I agree and please use our new syntax `> [!WARNING]`